src/ontology/ro.Makefile

```diff
@@ -53,9 +53,34 @@ $(IMPORTDIR)/cob_import.owl: $(MIRRORDIR)/cob.owl $(IMPORTDIR)/cob_terms_combine
 	if [ $(IMP) = true ]; then $(ROBOT) query -i $< --update ../sparql/preprocess-module.ru \
     extract -T $(IMPORTDIR)/cob_terms_combined.txt --copy-ontology-annotations true --force true --method BOT \
 	remove $(patsubst %, --term %, $(ANNOTATION_PROPERTIES)) -T $(IMPORTDIR)/cob_terms_combined.txt --select complement \
+	remove --select "RO:* BFO:0000050* BFO:0000051* BFO:0000060* BFO:0000066*" --axioms "annotation" \
 	query --update ../sparql/inject-subset-declaration.ru --update ../sparql/inject-synonymtype-declaration.ru --update ../sparql/postprocess-module.ru \
 	$(ANNOTATE_CONVERT_FILE); fi
 
+# ========================================
+# Custom imports
+# ========================================
+
+# This filter contains all property characteristic related to RO entities
+# Note: For some reason, IrreflexiveObjectProperty is not supported
+tmp/missing-base.owl: $(EDIT_PREPROCESSED) $(ONTOLOGYTERMS)
+	$(ROBOT) merge -i $(EDIT_PREPROCESSED) \
+		filter -T $(ONTOLOGYTERMS) --trim false --axioms "AsymmetricObjectProperty FunctionalObjectProperty FunctionalDataProperty InverseFunctionalObjectProperty SymmetricObjectProperty TransitiveObjectProperty InverseObjectProperties ObjectPropertyDomain DataPropertyDomain DataPropertyRange ObjectPropertyRange" \
+ -o $@.tmp.owl && mv $@.tmp.owl $@
+
+# TODO: This goal (and tmp/missing-base.owl) can be removed once https://github.com/ontodev/robot/issues/1108 is resolved
```

Comment:
No comment @anitacaron on this atrocity above :P At least you should say:

> @matentzn this is truly terrible. I don't have a better idea, but this is very much the worst hack I have seen in any Makefile. Shame on you.

cc @gouttegd who is usually more verbose in voicing his dismay at such... attrocities.

Reply:
Well, there is a comment with a link to an issue to give a clue as to why the hack is needed.

Not saying I _like_ that, but it’s far from being the worst thing I have ever seen in a Makefile.

Still, instead of linking to the issue on the ROBOT’s tracker it would have been better to create an issue _here_ to act as a reminder to remove the hack once it is no longer needed.

Reply:
Same thoughts 😄

Reply:
https://github.com/oborel/obo-relations/issues/794